packages/cli/src/resource/strings.json

```diff
@@ -1,34 +1,16 @@
 {
-  "account.login.azure": "You have successfully logged into Azure.",
-  "account.login.m365": "You have successfully logged into Microsoft 365.",
+  "account.login.azure": "Successfully signed into Azure.",
+  "account.login.azure.fail": "Failed to sign into Azure.",
+  "account.logout.azure": "Successfully signed out of Azure.",
+  "account.logout.azure.fail": "Failed to sign out of Azure.",
+  "account.login.m365": "Successfully signed into Microsoft 365.",
+  "account.login.m365.fail": "Failed to sign into Microsoft 365.",
+  "account.logout.m365": "Successfully signed out of Microsoft 365.",
+  "account.logout.m365.fail": "Failed to sign out of Microsoft 365.",
```

Comment:
Unable to sign out of Microsoft 365.

Reply:
Thanks for comments, I will create a new PR to fix this on the whole.